src/input.rs

```diff
@@ -303,6 +303,8 @@ pub enum TargetType {
 pub struct InputFile {
 	/// The name of the crate.
 	pub crate_name: String,
+	/// The version of the crate
+	pub crate_version: String,
```

Comment:
Why do we need this? The version of the crate should already be in the dependencies array

Reply:
Yes, but this does required the name of your crate. If you do rename your crate, you have to manual change all access to the array.
I think it is a good option to get the version of your crate by `crate_version`. Similar to `crate`, whitch provides the crate name.

Reply:
well at least we shouldn't convert it to a `String` here

Reply:
I don't know, but this struct is not the input for minijinja

Reply:
done